app/tailwind/config.js

```diff
@@ -25,8 +25,16 @@ module.exports = {
         'df-green': '#5fdc82',
         'df-green-dark': '#2bc456',
       },
+      zIndex: {
+        '-10': '-10',
```

Comment:
Is this being used? I couldn't find it...

Reply:
Oh woops that was when i was trying to use the divs and had to send them to the background. I can remove that easily